SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCloudLauncherSettings.h

```diff
@@ -0,0 +1,108 @@
+	FString AssemblyName;
+
+	UPROPERTY(EditAnywhere, config, Category = "General", meta = (ConfigRestartRequired = false, DisplayName = "Cloud launch configuration path"))
+	FFilePath PrimaryLaunchConfigPath;
+
+	UPROPERTY(EditAnywhere, config, Category = "General", meta = (ConfigRestartRequired = false, DisplayName = "Snapshot path"))
+	FFilePath SnapshotPath;
+
+	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (ConfigRestartRequired = false, DisplayName = "Include simulated players"))
+	bool bSimulatedPlayersIsEnabled;
+
+	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (EditCondition = "bSimulatedPlayersIsEnabled", ConfigRestartRequired = false, DisplayName = "Deployment mame"))
+	FString SimulatedPlayerDeploymentName;
+
+	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (EditCondition = "bSimulatedPlayersIsEnabled", ConfigRestartRequired = false, DisplayName = "Launch configuration path"))
+	FFilePath SimulatedPlayerLaunchConfigPath;
+
+	UPROPERTY(EditAnywhere, config, Category = "Simulated Players", meta = (EditCondition = "bSimulatedPlayersIsEnabled", ConfigRestartRequired = false, DisplayName = "Number of simulated players"))
+	uint32 NumberOfSimulatedPlayers;
+
+	static bool IsAssemblyNameValid(const FString & Name);
+	static bool IsProjectNameValid(const FString & Name);
+	static bool IsDeploymentNameValid(const FString & Name);
+
+public:
+	FString GetProjectNameFromSpatial() const;
+
+	void SetPrimaryDeploymentName(const FString & Name);
+	FORCEINLINE FString GetPrimaryDeploymentName() const
+	{
+		return PrimaryDeploymentName;
+	}
+
+	void SetAssemblyName(const FString & Name);
+	FORCEINLINE FString GetAssemblyName() const
+	{
+		return AssemblyName;
+	}
+
+	void SetProjectName(const FString & Name);
+	FORCEINLINE FString GetProjectName() const
+	{
+		return ProjectName;
+	}
+
+	void SetPrimaryLaunchConfigPath(const FString & Path);
+	FORCEINLINE FString GetPrimaryLanchConfigPath() const
+	{
+		return PrimaryLaunchConfigPath.FilePath;
+	}
+
+	void SetSnapshotPath(const FString & Path);
+	FORCEINLINE FString GetSnapshotPath() const
+	{
+		return SnapshotPath.FilePath;
+	}
+
+	void SetSimulatedPlayersEnabledState(bool IsEnabled);
+	FORCEINLINE bool IsSimulatedPlayersEnabled() const
+	{
+		return bSimulatedPlayersIsEnabled;
+	}
+
+	void SetSimulatedPlayerDeploymentName(const FString & Name);
+	FORCEINLINE FString GetSimulatedPlayerDeploymentName() const
+	{
+		return SimulatedPlayerDeploymentName;
+	}
+
+	void SetSimulatedPlayerLaunchConfigPath(const FString & Path);
+	FORCEINLINE FString GetSimulatedPlayerLaunchConfigPath() const
+	{
+		return SimulatedPlayerLaunchConfigPath.FilePath;
+	}
+
+	void SetNumberOfSimulatedPlayers(uint32 Number);
+	FORCEINLINE uint32 GetNumberOfSimulatedPlayer() const
+	{
+		return NumberOfSimulatedPlayers;
+	}
+
+	bool IsDeploymentConfigurationValid();
```

Comment:
Cant this be a const function now that the comparison functions are static?

Reply:
Made the function a const function.